packages/mco/components/create-dr-policy/add-s3-bucket-details/s3-bucket-details-form.tsx

```diff
@@ -0,0 +1,230 @@
+        isExpanded={expanded1}
+        onToggle={(_, exp) => setExpanded1(exp)}
+        isIndented
+      >
+        {fields.map(({ key, label }) => (
+          <FormGroup
+            key={key}
+            fieldId={`c1-${key}`}
+            label={label}
+            isRequired
+            className={inputPadding}
+          >
+            {key === 'secretKey' ? (
+              <div style={{ display: 'flex', alignItems: 'center' }}>
+                <TextInput
+                  type={show1 ? 'text' : 'password'}
+                  id={`c1-${key}`}
+                  value={cluster1Details[key]}
+                  placeholder={`Enter the ${label.toLowerCase()}`}
+                  onChange={(_, v) => update(1, key, v)}
+                  validated={errors1[key] ? 'error' : 'default'}
+                  style={{ flex: 1 }}
+                />
+                <Button
+                  variant="plain"
+                  onClick={() => setShow1((s) => !s)}
+                  aria-label="Toggle secret visibility"
+                >
+                  {show1 ? <EyeSlashIcon /> : <EyeIcon />}
+                </Button>
+              </div>
+            ) : (
+              <TextInput
+                id={`c1-${key}`}
+                value={cluster1Details[key]}
+                placeholder={`Enter the ${label.toLowerCase()}`}
+                onChange={(_, v) => update(1, key, v)}
+                validated={errors1[key] ? 'error' : 'default'}
+              />
+            )}
+            {errors1[key] && (
+              <HelperText>
+                <HelperTextItem variant="error">{errors1[key]}</HelperTextItem>
+              </HelperText>
+            )}
+          </FormGroup>
+        ))}
+      </ExpandableSection>
+      <Divider className="pf-v5-u-p-md" />
+      <ExpandableSection
+        toggleText={`S3 bucket for ${name2}`}
+        isExpanded={expanded2}
+        onToggle={(_, exp) => setExpanded2(exp)}
+        isIndented
+      >
+        <Checkbox
+          id="use-same-conn"
+          label="Use the same S3 connection details as the first cluster"
+          isChecked={useSameConnection}
+          onChange={onToggleSame}
+        />
+        {fields.map(({ key, label }) => (
+          <FormGroup
+            key={key}
+            fieldId={`c2-${key}`}
+            label={label}
+            isRequired
+            className={inputPadding}
+          >
+            {key === 'secretKey' ? (
+              <div style={{ display: 'flex', alignItems: 'center' }}>
+                <TextInput
+                  type={show2 ? 'text' : 'password'}
+                  id={`c2-${key}`}
+                  value={cluster2Details[key]}
+                  placeholder={`Enter the ${label.toLowerCase()}`}
+                  onChange={(_, v) => update(2, key, v)}
+                  validated={errors2[key] ? 'error' : 'default'}
+                  isDisabled={useSameConnection}
+                  style={{ flex: 1 }}
```

Comment:
1. Plz use either `scss` file or `pf-v5-` classes.
2. Try adding css directly to `TextInput`, why do we need `div` ??
```suggestion
                <TextInput
                  type={show2 ? 'text' : 'password'}
                  id={`c2-${key}`}
                  value={cluster2Details[key]}
                  placeholder={`Enter the ${label.toLowerCase()}`}
                  onChange={(_, v) => update(2, key, v)}
                  validated={errors2[key] ? 'error' : 'default'}
                  isDisabled={useSameConnection}
```